build_scripts/build_cfitsio.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -xe
+
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Cfitsio
+pushd third_party/cfitsio
```

Comment:
I would use CMake if possible.
Also please refer to https://github.com/aderylo/DALI_deps/blob/main/build_scripts/build_libflac.sh to see how to make sure that the cross compilation works as well.

Reply:
Ok, I've updated it to use CMake.